features/features.xml

```diff
@@ -358,6 +358,7 @@
     <test testName="text_experimental_obj_memcpy2d_api" configFile="config/TEMPLATE_image_excluding_syclcompat_and_cuda.xml" />
     <test testName="text_experimental_obj_memcpy3d_api" configFile="config/TEMPLATE_image_excluding_syclcompat.xml" />
     <test testName="text_experimental_obj_mipmap" configFile="config/TEMPLATE_image_excluding_syclcompat.xml" />
+    <test testName="text_experimental_tex_mipmap" configFile="config/TEMPLATE_image_excluding_syclcompat.xml" />
```

Comment:
Also need this test name to https://github.com/oneapi-src/SYCLomatic-test/blob/f21588b7b3871078d251eaf887e56b08672b662d/features/test_feature.py#L67 to enable run test for this case.